apps/examples/bluetooth/bt_unit_test.h

```diff
@@ -0,0 +1,606 @@
+/*
+ *
+ * Copyright (c) 2000 - 2011 Samsung Electronics Co., Ltd. All rights reserved.
```

Comment:
Please change this line to "  Copyright 2019 Samsung Electronics All Rights Reserved. "